pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -927,6 +927,11 @@ def main(args: argparse.Namespace) -> None:
     pkg.build.cxxflags += f" {args.cxxflags}"
     pkg.build.ldflags += f" {args.ldflags}"
 
+    missing_executables = find_missing_executables(pkg.requirements.executable)
+    if missing_executables:
+        error_msg = f"Following executables are required but missing in the host system: {', '.join(missing_executables)}"
```

Comment:
To keep the line a bit shorter maybe:
```suggestion
        missing_string = ", ".join(missing_executables)
        error_msg = (
            f"The following executables are required but missing in the host system: "
            + missing_string
        )
```